app/codecs/CirceDecoders.scala

```diff
@@ -82,7 +82,7 @@ object CirceDecoders {
   implicit val createPaymentMethodStateCodec: Codec[CreatePaymentMethodState] = deriveCodec
   implicit val failureHandlerStateCodec: Codec[FailureHandlerState] = deriveCodec
   implicit val completedStateCodec: Codec[CompletedState] = deriveCodec
-  implicit val switchStateEncode: Encoder[SwitchState] = Encoder.encodeString.contramap[SwitchState](s => s"'${s.toString}'")
+  implicit val switchStateEncode: Encoder[SwitchState] = Encoder.encodeString.contramap[SwitchState](s => s"${s.toString}")
```

Comment:
```scala
Encoder.encodeString.contramap[SwitchState](_.toString)
```

Reply:
not to mention that toString exists on everything, so should generally be avoided on anything other than displaying it to humans/logs

Reply:
`Show[SwitchState]` would be better but another time ✌️